stacks/upload-api-stack.js

```diff
@@ -28,6 +28,7 @@ export function UploadApiStack({ stack, app }) {
   const pkg = getApiPackageJson()
   const git = getGitInfo()
   const privateKey = new Config.Secret(stack, 'PRIVATE_KEY')
+  const uploadApiDid = new Config.Parameter(stack, 'UPLOAD_API_DID', { value: customDomain ? `did:web:${customDomain.domainName}` : '' })
```

Comment:
Perhaps add a comment what does this do ? I don't know much about SST and perhaps it would have been obvious if I knew. My guess is that this will either read from env variable or fallback to a thing with `customDomain` which I'm not sure what that one would be.

Does this imply we'll always have `uploadApiDid` ? Do we always want `did:web` would it make sense to stay with `did:key` if env var isn't set like you chose to do in access API ?

Reply:
@gozala this basically created a new parameter/environment variable 'UPLOAD_API_DID', and gives permissions to the upload-api lambdas to read it. The first time the stack and parameter are created, it'll set its initial value using `customDomain` if possible, otherwise it'll be empty string.

>  Do we always want `did:web` would it make sense to stay with `did:key` if env var isn't set like you chose to do in access API ?

If UPLOAD_API_DID gets set to empty string, I coded the rest so the ucanto server id signer would have a `did:key` as its did